MyCryptoHQ/MyCrypto
# Send Form Style Adjustments

Closes #1360, and a few unlogged odds and ends.

### Description

With all of the recent global changes that have happened, a few visual aspects of Send have gotten a little out of whack. This PR attempts to tighten all of them, and better set them up to not get messed up in the future.

This is definitely not the most important thing right now, but I felt paralyzed about changing other aspects of the send form until it was in a better state.

### Changes

* Turn "Send everything" link into button on the field (#1360)
* Remove "Account" tab title, lower change wallet button to align with content.
    * This was the only tab with a title, and it isn't even in the navigation anymore. It resulted in a ton of unnecessary whitespace.
* Change back address field label to "To Address" from "To"
    * I felt that the small label was easy to miss, and resulted in some awkward whitespace. I think the clarity here is worth the verbosity.
* Have the Request Payment tab use the full width of content, instead of 11/12 columns
* Fix AddressField identicon from hanging off of the content at smaller sizes
* Fix token balance button sizing / spacing
* Have WalletInfo use the AddressField component. AddressField now takes in a boolean prop `isSelfAddress` which uses the current wallet's address, instead of the current transaction's to address.

### Steps to Test

Browse around and check it out: https://mycryptobuilds.com/77ba75d7b1bb50ac8195af4a41b53fe30f6efb7d/

## Screenshots

### Full Page

![full-page](https://user-images.githubusercontent.com/649992/37788996-2fae2232-2dd9-11e8-8bff-8aefc8442746.png)


### Identicon

![identicon](https://user-images.githubusercontent.com/649992/37788990-2d0820fa-2dd9-11e8-84c0-651d16891380.png)


### Token Balances

![tokens](https://user-images.githubusercontent.com/649992/37788984-292310da-2dd9-11e8-8242-753bdab4e20a.png)

### Send Everything

<img width="172" alt="send-all-tooltip" src="https://user-images.githubusercontent.com/649992/37789074-5bc05e4e-2dd9-11e8-84b6-e4f64bde6edb.png">
